config/initializers/stripe.rb

```diff
@@ -0,0 +1,6 @@
+Rails.configuration.stripe = {
```

Comment:
I'm glad you commented on this Brandon. I was following a tutorial on the tealeaf blog when I created that. I ended up coming up with a different solution, but forgot to remove the stripe initializer. I'll take care of that now. Thanks!